paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlDebeziumJsonEventParser.java

```diff
@@ -382,4 +431,28 @@ private Map<String, String> keyCaseInsensitive(Map<String, String> origin) {
         }
         return keyCaseInsensitive;
     }
+
+    private boolean shouldExcludeCurrentTable() {
+        if (excludedTables.contains(currentTable)) {
+            return true;
+        }
+
+        boolean shouldInclude = true;
+        if (includingPattern != null) {
+            shouldInclude = includingPattern.matcher(currentTable).matches();
```

Comment:
NOTE this is executed per record.
We should have an `includeTables` to improve performance